docs/policy/campus-cyberinfrastructure.md

```diff
@@ -37,18 +38,20 @@ To contribute computational resources to the OSG, the following will be needed:
   worker node
 - SSH access to your local cluster's submit node from a known IP address
 - Shared home directories on each cluster node
+- Installation of some additional packages on the local cluster, IF the site would like to maximize it's ability to support
+  users, including those with large per-job data, containerized software, and/or GPU jobs.
 
 !!!success "Next steps"
-    If you are interested in OSG-offered services, please [contact us](mailto:help@opensciencegrid.org) for a
+    If you are interested in OSG-offered services, please [contact us](mailto:support@opensciencegrid.org) for a
     consultation, even if your site does not meet all the conditions as outlined above!
 
 Additional Materials
 --------------------
 
 If you are interested in learning more about the OSG and what it means to share resources via the OSG services, consider
-reviewing the following slides that were presented at the recent CC\* PI meeting:
+reviewing the following presentations from our [October 2020 workshop on dHTC and OSG services for campuses](https://indico.fnal.gov/event/45998/timetable/#20201022) ([YouTube Playlist](https://www.youtube.com/playlist?list=PLBWb4iScSWcPGfjvJztz_IeHCKS7f3u1k)):
 
-- [Introduction to OSG and OSG Participants](../files/Introduction-to-OSG.pdf)
-- [How Resources are Shared and Used via OSG](../files/OSG-Sharing-Resources.pdf)
-- [Contributing OSG Resources through Hosted CE](../files/Contributing-OSG-Resources.pdf)
-- [Using and Facilitating the Use of OSG Resources](../files/Using-OSG-Resources.pdf)
+- Intro to dHTC and PATh Services for Campuses ([slides](https://indico.fnal.gov/event/45998/contributions/199894/attachments/136489/169747/20201022OSGCampusWorkshop_dHTCServiceOverview_LMichael.pdf),[YouTube](https://www.youtube.com/watch?v=hYc6RaWL33g&list=PLBWb4iScSWcPGfjvJztz_IeHCKS7f3u1k&index=2&t=5s))
+- How OSG Works ([slides](https://indico.fnal.gov/event/45998/contributions/199896/attachments/136496/169757/ccstarOctober2020-whatisosg.pdf), [YouTube](https://www.youtube.com/watch?v=kwIs7oa56t8&list=PLBWb4iScSWcPGfjvJztz_IeHCKS7f3u1k&index=4)
+- Intro to OSG Resource Sharing ([slides](https://indico.fnal.gov/event/45998/contributions/199898/attachments/136497/169758/ccstarOctober2020-resourcesharing.pdf), [YouTube](https://www.youtube.com/watch?v=GuR8gxrVRew&list=PLBWb4iScSWcPGfjvJztz_IeHCKS7f3u1k&index=5))
+- Resource Sharing Technology, Security, System Requirements, Setup Process ([slides](https://indico.fnal.gov/event/45998/contributions/199903/attachments/136555/169835/OSG-Sharing-Resources-CCStar-2020.pdf), [YouTube](https://www.youtube.com/watch?v=t5YtIIs7bqg&list=PLBWb4iScSWcPGfjvJztz_IeHCKS7f3u1k&index=7))
```

Comment:
@lmichael107 Maybe we should store the slides in the repo as before? Probably not as strictly necessary as before with them being in Indico instead of Google Drive

Reply:
yeah, let's keep it all in one place (Indico), which we just didn't have before.